src/lib.rs

```diff
@@ -28,6 +28,8 @@ pub use logging::*;
 #[cfg(feature = "physics")]
 pub use crate::world::physics::*;
 
+pub use glam;
```

Comment:
good call! Re-exporting used libraries is really nice imo, prevents users from having to re-install them in their own crate.